features/core_min_bootstrap.feature

```diff
@@ -1,62 +1,62 @@
 # Copyright (c) 2016 SUSE LLC
 # Licensed under the terms of the MIT license.
 
-Feature: register a salt-minion via bootstrap
+Feature: Be able to bootstrap a Salt minion via the GUI
 
-  Scenario: Create bootstrap-repo for a salt client
+  Scenario: Create the bootstrap repository for a Salt client
      Given I am authorized
      And I create the "x86_64" bootstrap-repo for "sle-minion" on the server
 
-  Scenario: bootstrap a sles minion (it will be deleted after)
+  Scenario: Bootstrap a SLES minion that will be deleted after
      Given I am authorized
-     And I go to the bootstrapping page
+     When I go to the bootstrapping page
      Then I should see a "Bootstrap Minions" text
-     And  I enter the hostname of "sle-minion" as hostname
+     When I enter the hostname of "sle-minion" as hostname
      And I enter "22" as "port"
      And I enter "root" as "user"
      And I enter "linux" as "password"
      And I click on "Bootstrap"
-     Then I wait until i see "Successfully bootstrapped host! " text
+     Then I wait until I see "Successfully bootstrapped host! " text
 
-  Scenario: check new bootstrapped minion in System Overview page
+  Scenario: Check the new bootstrapped minion in System Overview page
      Given I am authorized
-     And I go to the minion onboarding page
+     When I go to the minion onboarding page
      Then I should see a "accepted" text
      And the salt-master can reach "sle-minion"
-     And I navigate to "rhn/systems/Overview.do" page
+     When I navigate to "rhn/systems/Overview.do" page
      And I wait until I see the name of "sle-minion", refreshing the page
      And I wait until onboarding is completed for "sle-minion"
 
-  Scenario: Run a remote command sles-minion (salt-service)
+  Scenario: Run a remote command on normal SLES minion
```

Comment:
is normal not too neutral?

Reply:
that's the way we speak in the office... normal = Salt service (not SSH nor reverse tunnel)